📝(project) add uploaded file and downloaded file statement templates (#16)
gaia-x-dases/xapi-common-profile: README.md

> # xAPI common profile for Open Education

Reviewer: ```suggestion
# xAPI LMS profile for Open Education
```

> This documentation intends to describe a xAPI common learning profile defined by the Gaia-X organisation.

Reviewer: ```suggestion
This documentation intends to describe LMS profile defined by the Gaia-X organisation.
```

> ## Common learning profile

Reviewer: ```suggestion
## Statements
```
